platform/migration/platform-migratable-api/src/main/java/org/codice/ddf/migration/MigrationMetadata.java

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p/>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p/>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.migration;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class MigrationMetadata {
+    
+    private List<MigrationWarning> migrationWarnings;
+
+    public MigrationMetadata() {
+        migrationWarnings = new ArrayList<>();
+    }
+    
+    public List<MigrationWarning> getMigrationWarnings() {
+        return migrationWarnings;
```

Comment:
Should we `return Collections.unmodifiableList(migrationWarnings)` instead to make sure the caller cannot change the list?